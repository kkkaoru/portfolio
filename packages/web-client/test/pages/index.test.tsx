import { render } from '@testing-library/react';
import Home from '@/pages/index';

describe('Top page', () => {
  it('should be matched snapshot', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
