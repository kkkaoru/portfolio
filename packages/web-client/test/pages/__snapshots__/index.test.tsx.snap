// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top page should be matched snapshot 1`] = `
<DocumentFragment>
  <div>
    <main>
      <div>
        <h1>
          <div
            class="flex flex-col items-center justify-center w-full p-10 font-bold break-all text-7xl lg:text-9xl after:block after:w-full after:h-1 after:bg-red-100"
          >
            Example nextjs, typescript, tailwindcss
          </div>
        </h1>
        <button
          class="font-sans font-bold border-0 rounded-[3em] cursor-pointer inline-block leading-none text-base py-[11px] px-[20px] text-[#333] bg-transparent shadow-md"
          type="button"
        >
          Example normal button
        </button>
        <button
          class="font-sans font-bold border-0 rounded-[3em] cursor-pointer inline-block leading-none text-base py-[11px] px-[20px] text-[#fff] bg-[#1ea7fd]"
          type="button"
        >
          Example primary button
        </button>
      </div>
      <div>
        <h2>
          Molecules Example
        </h2>
        <div>
          <div
            class="flex flex-col items-center justify-center w-full p-10 font-bold break-all text-7xl lg:text-9xl after:block after:w-full after:h-1 after:bg-red-100"
          >
            Molecules 
          </div>
          <button
            class="font-sans font-bold border-0 rounded-[3em] cursor-pointer inline-block leading-none text-base py-[11px] px-[20px] text-[#fff] bg-[#1ea7fd]"
            type="button"
          >
            Molecules button
          </button>
        </div>
      </div>
    </main>
    <footer>
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <div
          style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
        >
          <div
            style="box-sizing: border-box; display: block; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzIiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
              style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
            />
          </div>
          <noscript />
          <img
            alt="Vercel Logo"
            data-nimg="true"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </div>
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
